lib/matplotlib/backend_bases.py

```
@@ -3551,7 +3593,8 @@ def _key_press(self, event):
         name = self._keys.get(event.key, None)
         if name is None:
             return
-        self.tool_trigger_event(name, canvasevent=event)
+        try: self.tool_trigger_event(name, canvasevent=event)
```

I don't understand why this should fail silently?